api/db/models/blog.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const comment = Schema({
+  userId: {
+    required: true,
+    type: String,
+  },
+  content: {
+    required: true,
+    type: String,
+  },
+  createdAt: {
+    type: String,
+    type: Date,
+    default: Date.now,
+  },
+  updatedAt: {
+    type: String,
+    type: Date,
+    default: Date.now,
+  },
+});
+
+const post = Schema(
+  {
+    author: {
+      required: true,
+      type: String,
+    },
+    title: {
+      required: true,
+      type: String,
+    },
+    content: {
+      required: true,
+      type: String,
+    },
+    likes: {
+      type: [String],
+      default: [],
+    },
+    comments: {
+      ref: 'comment',
```

Comment:
Reference should correspond to entity name, stated in L53 as 'Comment'

Reply:
fixed